sys/include/ztimer.h

```diff
@@ -21,9 +21,10 @@
  * They all take a pointer to a clock device (or virtual timer device) as first
  * parameter.
  *
- * RIOT provides ZTIMER_USEC, ZTIMER_MSEC, ZTIMER_SEC by default, which can be
- * used in an application by depending on the modules ztimer_usec, ztimer_msec
- * or ztimer_sec. They will then automatically get configured.
+ * RIOT provides ZTIMER_NSEC, ZTIMER_USEC, ZTIMER_MSEC, ZTIMER_SEC by default,
```

Comment:
i am not sure if this  `default` applies that well to `NSEC` since there is just one board that supports this